### PR TITLE
Add svelte-credit-cards component package

### PR DESCRIPTION
**Pull request by CharlyJazz**
Hello there! This is my first package, please check it out

[Github Page](https://github.com/CharlyJazz/svelte-credit-card)

### CODE REVIEW

**Comment by swyxio** [comment]
Deploy Preview is done!

Built with commit 288a69b631da459c741dfec7090d5c7c3fd33303

https://deploy-preview-207--svelte-community.netlify.app

File: data/code.yml [hunk 1]
```diff
@@ -2038,3 +2038,11 @@ resources:
     stars: 6
     last_updated: '2020-05-11T18:35:41Z'
     downloads: 812
+  - name: '`svelte-credit-cards`'
+    url: 'https://github.com/CharlyJazz/svelte-credit-card'
+    description: A component to render a credit card preview, useful for ecommerces.
```
Comment (benmccann) [inline data/code.yml]: I might reword this as:

> A component to render a credit card preview - useful for ecommerce.

**Review by benmccann — COMMENTED** [review]
cool!

File: data/code.yml (hunk 1 above)
Comment (CharlyJazz) [reply data/code.yml]: Sure
